src/data/markdown/docs/01 guides/05 Testing Guides/02 Load testing websites.md

```diff
@@ -3,115 +3,99 @@ title: "Load testing websites"
 excerpt: "This guide provides some recommendations to help you load testing a website or web app with k6."
 ---
 
-This guide describes some recommendations to help you load testing a website or web app.
+Bad user experience impacts business. This is the reason you might want to know how many concurrent users your website can handle "well". 
 
-A load test of a website simulates many concurrent users mimicking one or several real user journeys that generally involve complex actions in a logical flow.  This guide can help you creating these user scenarios, getting started, and avoid common pitfalls.
+That is what load testing is all about. Load testing will help you simulate those users to asses your system performance and know if your website or web app meet your goals.
 
-## Should I load test my website?
+This guide will provide guidance and recommendations to get started, creating user scenarios, and avoiding common pitfalls to load test a website.
 
-You should if you are concerned about the <b>availability and scalability of your website</b>.
+## What does affect the performance of a website?
```

Comment:
`What does affect the performance of a website?` -> `Factors that affect website performance`